README.md

```diff
@@ -2,117 +2,40 @@
 
 Réduire le nombre de rendez-vous annulés dans les maisons départementales de solidarité.
 
-https://beta.gouv.fr/startups/lapins.html
+## Présentation
 
-[![View performance data on Skylight](https://badges.skylight.io/status/RgR7i58P67xN.svg)](https://oss.skylight.io/app/applications/RgR7i58P67xN)
-
-## Installation pour le développement
-
-### Dépendances techniques
-
-#### Tous environnements
-
-- Ruby 2.7.1
-- postgresql
-
-#### Développement
-
-- rbenv : voir https://github.com/rbenv/rbenv-installer#rbenv-installer--doctor-scripts
-- Yarn : voir https://yarnpkg.com/en/docs/install
-- Foreman : voir https://github.com/ddollar/foreman
-
-#### Tests
-
-- Chrome
-- chromedriver :
-  * Mac : `brew cask install chromedriver`
-  * Linux : voir https://sites.google.com/a/chromium.org/chromedriver/downloads
-
-### Initialisation de l'environnement de développement
-
-Afin d'initialiser l'environnement de développement, exécutez la commande suivante :
-
-```bash
-bin/setup
-```
-Vous pouvez aussi vous créer un compte SuperAdmin dans une console Rails :
+### Une solution numérique pour faciliter la prise, l’annulation et le remplacement de rendez-vous
 
-```
-SuperAdmin.create!(email: 'email_associated_to_your_github_account@prov.com')
-```
+L'application permet de fluidifier la prise de rendez-vous. Elle permet la simplification du travail d'organisation et de suivi par les agents. Cinq moyens de base sont identifiés : la saisie unique des informations de l'usager, l'usage d'agendas partagés,  des rappels automatiques par email et SMS. Dans certains cas, les usagers peuvent prendre rendez-vous en ligne et modifier ou annuler leur rendez-vous en autonomie. Des files d'attente pour prévenir automatiquement les usagers de créneaux rendus disponibles sont mis en place. 
```

Comment:
est-ce que ces contenus ont été rédigées en accord / en doublon de la doc ? Je pense qu'il faut plutôt pencher pour de la duplication que des contenus proches mais différents ici. 

Le mieux à mon avis serait donc d'avoir les textes "originaux" sur doc.rdv-solidarites.fr , et ici de dupliquer ceux qu'on souhaite faire apparaître tout de suite dans le README du repo, juste les principaux, et rajouter un lien vers la doc pour plus de détails. La redondance est "nécessaire" ici pour afficher l'info au plus proche, mais il vaut mieux la limiter pour éviter les divergences futures inévitables

Reply:
Nous avons repris effectivement ce qu'il y avait d'écrit dans la Doc de Lapin. Je me suis mise un pot-it pour faire la modification sur les site de Laipn de Beta Gouv pour lui apporter les modification nécessaire. (histoire que tout soit à jour ;) !.)

Reply:
J'ai donc indiquer les liens suivant pour les conventions : https://github.com/rubocop-hq/rails-style-guide ==> pour les conventions d'écriture (je me demande si cela peu être mis dans Rubocop par défaut ?) 

et j'ai mis : https://rails-bestpractices.com/ => qui permet de savoir quel outil de Rail (méthode) est mieux à utilise et pourquoi / ou pourquoi faire comme ça avec Rails.